Documentation/ApiOverview/Hooks/Events/Backend/AfterPageColumnsSelectedForLocalizationEvent.rst

```diff
@@ -0,0 +1,84 @@
+.. include:: ../../../../Includes.txt
+
+
+.. _AfterPageColumnsSelectedForLocalizationEvent:
+
+
+============================================
+AfterPageColumnsSelectedForLocalizationEvent
+============================================
+
+Event to listen to after the form engine has been initialized (= all data has been persisted).
```

Comment:
Should `(=` be `and` here? I wouldn't expect the Form Engine to persist data on init already ...